File: src/js/pe_0009.js
// FUNCTIONS


function is_triplet(a, b, c) {
    return (a < b < c) && (a * a + b * b == c * c)
}


// SCRIPT


for (let a = 1; a < 999; a++) {
    for (let b = a + 1; b < (1000 - a); b++) {
        let c = 1000 - (a + b);
        if (is_triplet(a, b, c)) {
            console.log("%d %d %d", a, b, c);
            process.exit();
        }
    }
}

File: src/py/pe_0009.py
# FUNCTION


def is_triplet(a, b, c):
    return (a < b < c) and (a ** 2 + b ** 2 == c ** 2)


# SCRIPT


for a in range(1, 1000):
    for b in range(a + 1, 1000 - a):
        c = 1000 - (a + b)
        if is_triplet(a, b, c):
            print(a, b, c)
            exit()
